api/swagger.yaml

```diff
@@ -8343,6 +8343,16 @@ paths:
           description: "server error"
           schema:
             $ref: "#/definitions/ErrorResponse"
+      parameters:
+        - name: "types"
+          in: "query"
+          description: |
+            Object types, for which to compute and return data. For example, `containers,images`.
+          type: "array"
+          uniqueItems: true
```

Comment:
Looks like this is not "strictly" needed (as we are able to handle duplicates). Guess it doesn't  _hurt_, but not strictly "true" 😅